src/main/java/org/opensearch/flowframework/rest/RestGetWorkflowAction.java

```diff
@@ -88,7 +92,7 @@ protected BaseRestHandler.RestChannelConsumer prepareRequest(RestRequest request
                     channel.sendResponse(new BytesRestResponse(ex.getRestStatus(), exceptionBuilder));
 
                 } catch (IOException e) {
-                    logger.error("Failed to send back provision workflow exception", e);
+                    logger.error("Failed to send back get template exception", e);
```

Comment:
the GET api path looks like this: `_plugins/_flow_framework/workflow` , it would make more sense if we change this to "Failed to send back get workflow exception"? could be weird to do a GET operation on a workflow ID under `/workflow/` and see template in exception message

Reply:
sure ill make this change